tests/testthat/_snaps/survival-augment.md

```diff
@@ -1,8 +1,8 @@
-# augment() for survival models errors if eval_time is missing
+# augment() works for tune_results
 
     Code
-      augment(sr_fit, lung)
+      aug_res <- augment(grid_mixed_res)
     Condition
-      Error in `augment()`:
-      ! The `eval_time` argument is missing, with no default.
+      Warning in `show_best()`:
+      4 evaluation times are available; the first will be used (i.e. `eval_time = 10`).
```

Comment:
Does this seem right? Should we modify `augment.tune_results()` such that this doesn't happen. There are currently no way of avoiding this as `augment.tune_results()` doesn't take the arguments it need

Reply:
That's https://github.com/tidymodels/tune/issues/810 

We want `augment()` to treat `eval_time` like it treats `metric`: if you don't specify which parameters you want to use, it will use the first metric and pass that to `show_best()`. Analogous, it should pick the first eval_time and pass that. Both parts documented, of course.

Reply:
got it! `augment.tune_results()` doesn't have a `metric` argument, instead it has a `parameters` argument. These are now tested with [`fa2099e` (#178)](https://github.com/tidymodels/extratests/pull/178/commits/fa2099e2e4ef6898a45a7039f19b295f577db674)